crates/nix_rs/src/flake/eval.rs

```diff
@@ -13,10 +13,13 @@ where
     T: serde::de::DeserializeOwned,
 {
     let stdout = nixcmd
-        .run_with_returning_stdout(|cmd| {
+        .trace_run_with_returning_stdout(|cmd| {
             cmd.args(["eval", "--json"]);
             opts.use_in_command(cmd);
             cmd.arg(url.to_string());
+            // Decrease logging verbosity to avoid polluting the `om` output
+            // with logs other than the flake evaluation progress.
+            cmd.args(["--quiet", "--quiet"]);
```

Comment:
Double `--quiet`?

Reply:
Okay, so it is a repeating option? In that case, `-qq` would be clearer as that's what people normally type on the terminal (like `-vv` for instance).

Reply:
Also, the second quiet level hides "Added input" output? That's interesting; I wonder if we can do that for other commands (not in this PR).

Reply:
> Okay, so it is a repeating option? In that case, -qq would be clearer 

unlike `—verbose` there is no shorthand for `—quiet`. From `nix eval —help`:

```sh
      · --quiet Decrease the logging verbosity level.

      · --verbose / -v Increase the logging verbosity level.
```

Reply:
> Also, the second quiet level hides "Added input" output? That's interesting; I wonder if we can do that for other commands (not in this PR).

Yes, it does

Reply:
> unlike `--verbose` there is no shorthand for `--quiet`.

Alright then. Sounds like an idea for a PR to nix repo.

Reply:
https://github.com/NixOS/nix/pull/11652